packages/docs-app/src/examples/core-examples/popoverInlineExample.tsx

```diff
@@ -20,38 +28,53 @@ export class PopoverInlineExample extends BaseExample<{}> {
     };
 
     public componentDidMount() {
-        this.recenter();
+        // if we don't requestAnimationFrame, this function apparently executes
+        // before styles are applied to the page, so the centering is way off.
+        requestAnimationFrame(this.recenter);
+        // likewise, Popper.js doesn't handle controlled, non-inline popovers
+        // properly if rendered with isOpen={true} on initial mount: popovers
+        // won't follow the target when scrolled. to fix, defer opening.
```

Comment:
can this be done inside `Popover` itself? or maybe inside `Portal`, which now tracks a `hasMounted` state field? 

seems like useful behavior that will be desirable across the board.

Reply:
I'm hesitant to make controlled Popover and/or Portal rendering asynchronous...

Can I file an issue to track this? Would prefer to see how things fare in the wild as is—then return if it proves painful for more folks than just me.

Reply:
Either way, I'll merge this PR now and follow up somehow after.